Automattic/wp-calypso
# Move email management components

#### Changes proposed in this Pull Request

* Move components client/my-sites/domains/domain-management/email => client/my-sites/email/email-management

#### Testing instructions

* Goto the email tab for domain with G Suite
* Observe users are rendered properly
* Goto the email tab for domain w/o G Suite
* Observe that G Suite cta appears
